Cython/Includes/cpython/descr.pxd

```diff
@@ -0,0 +1,26 @@
+from .object cimport PyObject, PyTypeObject
+
+cdef extern from "Python.h":
+    ctypedef object (*wrapperfunc)(self, args, void* wrapped)
+    ctypedef object (*wrapperfunc_kwds)(self, args, void* wrapped, kwds)
+
+    struct wrapperbase:
+        char* name
+        int offset
+        void* function
+        wrapperfunc wrapper
+        char* doc
+        int flags
+        PyObject* name_strobj
+
+    int PyWrapperFlag_KEYWORDS
+
+    ctypedef class __builtin__.wrapper_descriptor [object PyWrapperDescrObject]:
+        cdef type d_type
+        cdef d_name
```

Comment:
This is not portable as `d_type` and `d_name` are directly Python 2 specific, while Python 3 defines them indirectly through the `PyDescrObject` member `d_common`.

It's true that this could be addressed right in this definition file but I'm of the opinion that it doesn't worth it—Python, regardless of the version, already exposes them as attributes, named `__objclass__` and `__name__`, respectively;  it'd end being somewhat verbose, especially if other builtin descriptor objects get ever defined here;  and whatever internals a user may care about will likely be already defined (e.g. `d_base` and `d_wrapped` in this case).

While I'm on it, may I request the inclusion of a definition for the `getset_descriptor` type as well?